jenkins/internal-ci/generic/pr/cortx_deployment/pyutils.groovy

```diff
@@ -0,0 +1,252 @@
+        CORTX_ISO_LOCATION = "${DESTINATION_RELEASE_LOCATION}/cortx_iso"
+        THIRD_PARTY_LOCATION = "${DESTINATION_RELEASE_LOCATION}/3rd_party"
+        PYTHON_LIB_LOCATION = "${DESTINATION_RELEASE_LOCATION}/python_deps"
+		
+        build_id = sh(script: "echo ${CORTX_BUILD} | rev | cut -d '/' -f2,3 | rev", returnStdout: true).trim()
+        STAGE_DEPLOY = "yes"
+    }
+
+    stages {
+
+        // Build py_utils fromm PR source code
+        stage('Build') {
+            steps {
+				script { build_stage = env.STAGE_NAME }
+                script { manager.addHtmlBadge("&emsp;<b>Target Branch : ${BRANCH}</b>&emsp;<br />") }
+
+                sh """
+                    set +x
+                    echo "--------------BUILD PARAMETERS -------------------"
+                    echo "PY_UTILS_URL              = ${PY_UTILS_URL}"
+                    echo "PY_UTILS_BRANCH           = ${PY_UTILS_BRANCH}"
+                    echo "PY_UTILS_PR_REFSPEC       = ${PY_UTILS_PR_REFSPEC}"
+                    echo "-----------------------------------------------------------"
+                """
+                 
+                dir("cortx_utils") {
+
+                    checkout([$class: 'GitSCM', branches: [[name: "${PY_UTILS_BRANCH}"]], doGenerateSubmoduleConfigurations: false, extensions: [[$class: 'AuthorInChangelog']], submoduleCfg: [], userRemoteConfigs: [[credentialsId: 'cortx-admin-github', url: "${PY_UTILS_URL}",  name: 'origin', refspec: "${PY_UTILS_PR_REFSPEC}"]]])
+
+                    sh label: 'Build', script: '''
+                        yum install python36-devel -y
+                        ./jenkins/build.sh -v $VERSION -b $BUILD_NUMBER
+                        ./statsd-utils/jenkins/build.sh -v $VERSION -b $BUILD_NUMBER
+                    '''
+                }
+            }
+        }
+
+        // Release cortx deployment stack
+        stage('Release') {
+            steps {
+				script { build_stage = env.STAGE_NAME }
+                echo "Creating Provisioner Release"
+                dir('cortx-re') {
+                    checkout([$class: 'GitSCM', branches: [[name: '*/main']], doGenerateSubmoduleConfigurations: false, extensions: [[$class: 'CloneOption', depth: 1, honorRefspec: true, noTags: true, reference: '', shallow: true], [$class: 'AuthorInChangelog']], submoduleCfg: [], userRemoteConfigs: [[credentialsId: 'cortx-admin-github', url: 'https://github.com/Seagate/cortx-re']]])
+                }
+
+               // Install tools required for release process
+                sh label: 'Installed Dependecies', script: '''
+                    yum install -y expect rpm-sign rng-tools python3-pip
+                    #if [ "${OS_FAMILY}" == "rockylinux" ]
+                    #then
+                    #    ln -fs $(which python3.6) /usr/bin/python2
+                    #else
+                    #    echo "Using CentOS"
+                    #fi
+                    #systemctl start rngd
```

Comment:
remove if code is not needed

Reply:
We can remove the commented code but without this we can build rockylinux image without sign the rpm  but we are not sure about the centos part that's why I keep this commented part

Reply:
Okay.